src/Serialization/Json/OkContentNormalizer.php

```diff
@@ -0,0 +1,49 @@
+<?php
+
+namespace Biig\Melodiia\Serialization\Json;
+
+use Biig\Melodiia\Response\OkContent;
+use Symfony\Component\Serializer\Normalizer\NormalizerInterface;
+
+/**
+ * Class OkContentNormalizer.
+ *
+ * Ignore the OkContent class and return the normalization of its content.
```

Comment:
Normalize the content of `OkContent` class

Reply:
I wanted to point that it kinda "delete" the "OkContent" level at normalization. (and it's pretty much its only job)

Reply:
Fixed